benchmarks/2019.json

```diff
@@ -1511,24 +1511,6 @@
       100
     ]
   },
-  {
-    "measureId": "128",
```

Comment:
Are we ok w/o having '128' w/ submissionMethod: cmsWebInterface?

Reply:
Yes. Natalia did a very comprehensive analysis of the cmsWebInterface changes for Y3. See here for more details: https://confluence.cms.gov/pages/viewpage.action?spaceKey=QPPPRACTICE&title=Year+3+Changes+in+Web+Interface